include/veriblock/blockchain/alt_block_tree.hpp

```diff
@@ -133,7 +148,9 @@ struct AltTree : public BaseBlockTree<AltBlock> {
 
   index_t* insertBlockHeader(const AltBlock& block);
 
-  void payloadsToCommands(const payloads_t& p,
+  void payloadsToCommands(const std::vector<alt_payloads_t>& alt_payloads,
+                          const std::vector<vbk_payloads_t>& vbk_payloads,
+                          const std::vector<VbkBlock>& context,
                           std::vector<CommandPtr>& commands);
```

Comment:
This interface should be more generic as PopStateMachine uses this interface. Eg VbkBlockTree is using the same interface for it's payloads.